dev/TOC.md

```diff
@@ -340,19 +306,54 @@
     - [TiKV Control](reference/tools/tikv-control.md)
     - [TiDB Controller](reference/tools/tidb-control.md)
     - [工具下载](reference/tools/download.md)
++ TiDB in Kubernetes
+  - [TiDB Operator 简介](tidb-in-kubernetes/tidb-operator-overview.md)
+  + 快速上手
+    - [DinD](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-dind.md)
+    - [GKE](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-gke.md)
+    - [Minikube](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-minikube.md)
+  + 部署
+    - [集群环境要求](tidb-in-kubernetes/deploy/prerequisites.md)
+    - [部署 TiDB Operator](tidb-in-kubernetes/deploy/tidb-operator.md)
+    - [访问 Kubernetes 上的 TiDB 集群](tidb-in-kubernetes/deploy/access-tidb.md)
+    - [标准 Kubernetes 上的 TiDB](tidb-in-kubernetes/deploy/general-kubernetes.md)
+    - [AWS EKS 上的 TiDB](tidb-in-kubernetes/deploy/aws-eks.md)
+    - [GCP 上的 TiDB](tidb-in-kubernetes/deploy/gcp-gke.md)
+    - [Alibaba Cloud 上的 TiDB](tidb-in-kubernetes/deploy/alibaba-cloud.md)
+  + 配置
+    - [初始化集群](tidb-in-kubernetes/initialize-cluster.md)
+  - [监控](tidb-in-kubernetes/monitor/tidb-in-kubernetes.md)
+  + 运维
+    - [销毁 TiDB 集群](tidb-in-kubernetes/maintain/delete-tidb-cluster.md)
```

Comment:
```suggestion
    - [销毁 TiDB 集群](tidb-in-kubernetes/maintain/destroy-cluster.md)
```

Reply:
Will change to `detroy-tidb-cluster` to be consistent with docs.

Reply:
That's good.